tutorials/remote-staking-xcm.md

```diff
@@ -0,0 +1,246 @@
+
+For development purposes this tutorial is written for Moonbase Alpha and Moonbase relay using TestNet funds. For prerequisites:
+
+* You should have DEV tokens which you can acquire from the [Moonbase Alpha faucet](https://apps.moonbeam.network/moonbase-alpha/faucet/){target=_blank} 
+* You should also have some UNIT, the native token of the Moonbase relay chain. You can swap some DEV for xcUNIT here on [Moonbeam Swap](https://moonbeam-swap.netlify.app/#/swap){target=_blank} 
+* Lastly, you'll need to withdraw the xcUNIT from Moonbase Alpha to [your account on the Moonbase relay chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/accounts){target=_blank} using [apps.moonbeam.network](https://apps.moonbeam.network/moonbase-alpha/){target=_blank}. 
+
+Copy the account of your existing or newly created account on the [Moonbase relay chain](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Ffrag-moonbase-relay-rpc-ws.g.moonbase.moonbeam.network#/accounts){target=_blank}. You're going to need it to calculate the corresponding multilocation derivative account, which is a special type of account that’s keyless (the private key is unknown). Transactions from a multilocation derivative account can be initiated only via valid XCM instructions from the corresponding account on the relay chain. In other words, you are the only one who can initiate transactions on your multilocation derivative account - and if you lose access to your Moonbase relay account, you’ll also lose access to your multilocation derivative account. 
+
+To generate the multilocation derivative account, first clone Alberto’s [xcmTools repo](https://github.com/albertov19/xcmTools){target=_blank}. Run `yarn` to install the necessary packages and then run:
+
+
+    ts-node calculateMultilocationDerivative.ts \
+     --w wss://wss.api.moonbase.moonbeam.network \
+     --a YOUR-MOONBASE-RELAY-ACCOUNT-HERE \
+     --n 0x57657374656e64
+
+Let's review the parameters passed along with this command:
+
+* The `-w` flag corresponds to the endpoint we’re using to fetch this information
+* The `-a` flag corresponds to your Moonbase relay chain address
+* The `-n` flag corresponds to the encoded form of “westend”, the name of the relay chain that Moonbase relay is based on
+
+The script will return 32-byte and 20-byte addresses. We’re interested in the ethereum-style account - the 20-byte one. Feel free to look up your multilocation derivative account on [Moonscan](https://moonbase.moonscan.io/){target=_blank}. You’ll note that this account is empty. You’ll now need to fund this account with at least 1.1 DEV. As this is the amount that the faucet dispenses, you'll need to make a minimum of two faucet requests or you can always reach out to us on [Discord](https://discord.com/invite/amTRXQ9ZpW){target=_blank} for additional DEV tokens.
+
+## Preparing to Stake on Moonbase Alpha {: #preparing-to-stake-on-moonbase-alpha }
```

Comment:
Some of the headers are "Preparing" and others are just "Get", I would stick to "Prepare" and "Get" or "Preparing" and "Getting"